src/gui/tray/usermodel.cpp

```diff
@@ -505,7 +501,8 @@ void User::slotAddErrorToGui(const QString &folderAlias, SyncFileItem::Status st
         // add 'other errors' to activity list
         _activityModel->addErrorToActivityList(activity);
 
-        showDesktopNotification(activity._subject, activity._message);
+        // Error notifications don't have ids by themselves so we will create one for it
+        showDesktopNotification(activity._subject, activity._message, qHash(activity._subject + activity._message));
```

Comment:
I am wondering if collisions could be possible
there is also the option of asking the server teams to add the missing ids